lib/tasks/publishing_api.rake

```diff
@@ -0,0 +1,50 @@
+namespace :publishing_api do
+  desc "Publish special routes such as the homepage"
+  task :publish_special_routes => :environment do
+    require 'gds_api/publishing_api/special_route_publisher'
+
+    publisher = GdsApi::PublishingApi::SpecialRoutePublisher.new
+
+    routes = [
+      {
+        content_id: "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a",
+        base_path: "/",
+        title: "GOV.UK homepage",
+      },
+      {
+        content_id: "ffcd9054-ee77-4539-978d-171a60eb4b2a",
```

Comment:
This should have a base_path of `/homepage`.  Additionally I wondwe whether this would be better being created as a redirect item int he content-store.  Then we could remove the handling of this from the frontend application.